src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -53,6 +57,7 @@ internal static readonly HashSet<(string, string)> DefaultHistogramBoundLongMapp
         ("System.Net.Http", "http.client.connection.duration"),
     };
 
+    internal volatile int Status;
```

Comment:
Why not use an `enum`?

Reply:
Ended up removing the status and it went back to a bool. Originally I was thinking volatile wouldn't work with an enum but it actually seems to work fine. Good to know!